src/mtype.d

```diff
@@ -3492,7 +3492,7 @@ public:
             case Tcomplex80:
             case Timaginary80:
             case Tfloat80:
-                fvalue = Port.ldbl_max;
+                fvalue = real.max;
```

Comment:
DMD can't cross compile, which is why this is unconditional for them.  GDC has it's own implementation of ``port.c`` that queries the gcc backend for the correct value.